### DISCUSSION TITLE
Is it possible to connect a github website to a pterodactyl panel?

### DISCUSSION THREAD

**Discussion by TitaniumTries** [pterodactyl/panel#3859] [Support]
Like, everytime you make a change in github, it automatically gets changed in the pterodactyl server? If this is possibe, how is it done? Please send me an article or a yt tutorial to do this, many thanks!

**Comment by trenutoo** [comment]
You can make your egg/server git pull on each startup, similar to the generic language eggs such as [discord.js](https://github.com/parkervcp/eggs/tree/master/bots/discord/discord.js) do it as long as the Docker image has git installed.

To force restart server each time a commit is pushed, you need monitoring to which Github can send a webhook
